Exercise_recursive/recursive-array.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+# This script submits an array job on a single node with the 
```

Comment:
same here as my comment on the Exercise_nonsequential about putting the `#SBATCH` directive right after `#!/bin/bash`.